### DISCUSSION TITLE
Knowldege distillation

### DISCUSSION THREAD

**Discussion by Leila-Malihi** [yoshitomo-matsubara/torchdistill#363] [Q&A]
Hi,
Could you please tell me this algorithm you implemented for knowledge distillation is which one? I mean there is some like: Fitnet,AT, FT,..

**Comment by yoshitomo-matsubara** [comment]
Hi @Leila-Malihi ,

If you're asking about what method is specifically referred to as "Knowledge Distillation", it's [Hinton et al. (2014)](https://arxiv.org/abs/1503.02531)'s knowledge distillation (KD).
FitNet, AT, FT, and more also fall into the category of knowledge distillation

Close this discussion and "Mark as answer" if it answers your question
